changelog.md

```diff
@@ -0,0 +1,3 @@
+# azhpc Change log
+
+* PR#156 - Added permission option to sasurl variable value. Append ",rw" for read write access. Default is read.
```

Comment:
best to switch to the date here to make simpler

Reply:
switch to date now